### PR TITLE
Add JWT ACL description

### PR DESCRIPTION
**Pull request by ChrisGuzman**
## Description

Adding a clearer description of JWTs and ACLs

## Deploy Notes

This PR relies on #800. It shouldn't be merged until after then.

### CODE REVIEW

**Comment by ChrisGuzman** [comment]
Hmrm, maybe the "Nexmo Client Libraries" section can be tabbed content like in the VAPI building blocks?

**Comment by ericgiannini** [comment]
@ChrisGuzman I think the more granular / modular, the easier it is us for us to respond in situations of support.

File: _documentation/stitch/concepts/jwt-acl.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+      "/v1/sessions/**": {},
+      "/v1/devices/**": {},
+      "/v1/image/**": {},
+      "/v3/media/**": {},
+      "/v1/applications/**": {},
+      "/v1/push/**": {},
+      "/v1/knocking/**": {}
+    }
+  },
+  "application_id": "8ab31eef-e181-401e-a6b7-c2ca33a8a01c"
```
Comment (AlexLakatos) [inline _documentation/stitch/concepts/jwt-acl.md]: can we use the app id as `aaaaaaaa-bbbb-cccc-dddd-0123456789ab` everywhere please? same as in the "create application" command

File: _documentation/stitch/concepts/jwt-acl.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+It is not necessary to provide users with the permissions to access all of these paths. For instance if a user was not going to upload or receive push notifications, you could create a JWT without including the `/v1/applications/**`or `/v1/push/**` paths.
+
+## Nexmo Client Libraries
+
+### CLI
+
+Currently you can use the beta version of the [Nexmo CLI](https://github.com/Nexmo/nexmo-cli/tree/beta) to create a JWT [including the appropriate claims](https://github.com/Nexmo/nexmo-cli/tree/beta#jwt)
+
+```sh
+> nexmo jwt:generate path/to/private.key subject=username iat=1475861732 application_id=asdasdas-asdd-2344-2344-asdasdasd345
```
Comment (AlexLakatos) [inline _documentation/stitch/concepts/jwt-acl.md]: this doesn't need an iat

Comment (AlexLakatos) [reply _documentation/stitch/concepts/jwt-acl.md]: and the subject should be sub iirc

File: _documentation/stitch/concepts/jwt-acl.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+nexmo app:create "My Stitch App" https://example.com/answer https://example.com/event --type=rtc --keyfile=private.key
+```
+
+```
+Application created: aaaaaaaa-bbbb-cccc-dddd-0123456789ab
+No existing config found. Writing to new file.
+Credentials written to /path/to/your/local/folder/.nexmo-app
+Private Key saved to: private.key
+```
+
```
Comment (ericgiannini) [inline _documentation/stitch/concepts/jwt-acl.md]: Should we post about other ways to create a new application? https://dashboard.nexmo.com/voice/create-application

File: _documentation/stitch/concepts/jwt-acl.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+### Claims
+
+Using that `private.key` and the application ID. we can mint a new JWT. In order to log a user in to a Stitch client, the JWT will need the following claims:
```
Comment (ericgiannini) [inline _documentation/stitch/concepts/jwt-acl.md]: Should the `.` here be `,`?

File: _documentation/stitch/concepts/jwt-acl.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+- `sub`: The "subject". The subject in this case will be the name of the user created in Stitch
```
Comment (ericgiannini) [inline _documentation/stitch/concepts/jwt-acl.md]: Should we have a table like we do on the VAPI docs for statuses? https://developer.nexmo.com/voice/voice-api/guides/call-flow

File: _documentation/stitch/concepts/jwt-acl.md (hunk 2 above)
Comment (ChrisGuzman) [reply _documentation/stitch/concepts/jwt-acl.md]: It's also missing the ACL.

File: _documentation/stitch/concepts/jwt-acl.md (hunk 3 above)
Comment (ChrisGuzman) [reply _documentation/stitch/concepts/jwt-acl.md]: Yes good idea

File: _documentation/stitch/concepts/jwt-acl.md (hunk 5 above)
Comment (ChrisGuzman) [reply _documentation/stitch/concepts/jwt-acl.md]: yes good idea

**Comment by ericgiannini** [comment]
It looks like we are good to go.

File: _documentation/stitch/concepts/jwt-acl.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+## ACLs
+
+### Overview
+
+In the previous section, you can see that the `acl` claim has `paths` object containing multiple endpoints. These endpoints correspond with certain permissions a user can have when utilizing Stitch features.
+
+### Paths
+
+|Endpoint | Description |
```
Comment (leggetter) [inline _documentation/stitch/concepts/jwt-acl.md]: What does a recommended and sufficiently secure ACL look like a client-side JWT?

Comment (ChrisGuzman) [reply _documentation/stitch/concepts/jwt-acl.md]: That's a good question. I'll connect with @mariatechmaniac and @nstratford for proper documentation

**Pull request merged**